src/content/en/tools/workbox/guides/route-requests.md

```diff
@@ -38,7 +38,7 @@ workbox.routing.registerRoute(
 );
 ```
 
-The only thing to be weary of is that this would only match for requests
+The only thing to be wary of is that this would only match for requests
 on your origin. If there was a separate site what had the URL
```

Comment:
Should “what” also be changed to “that” or “which”?

Reply:
It should be 'that'.

Reply:
Thanks, done :D